ampeg/src/test/java/DriverManagerFactory.java

```diff
@@ -0,0 +1,20 @@
+public class DriverManagerFactory {
+	
+	public static DriverManager getManager(String browserType) {
+
+		DriverManager driverManager = null;
+
+		switch(browserType) {
+		case "CHROME":
+			driverManager = new ChromeDriverManager();
+			break;
+		case "EDGE":
+			driverManager = new EdgeDriverManager();
+			break;
+		default:
+			break;
```

Comment:
Throw an exception if an unsupported browserType is passed.
This will be the default.